main.js

```diff
@@ -1,52 +1,47 @@
-let user = 'John Doe';
-console.log(user);
-
-const student = 'Alex';
-console.log(student);
-
-user = student;
-console.log(user);
-
-console.log('-----');
-
-let test;
-test = 1;
-console.log(test);
-test++;
-console.log(test);
-test += '1';
-console.log(test);
-test--;
-console.log(test);
-test = true;
-console.log(test);
+const pali = 'шалаш';
+console.log(pali);
+
```

Comment:
Your function doesn't work correctly, chec it please, in you inplementation it's chec only three letter in the end and at the start, but if other letter will not equal  it return true. So you can make it in loop (cycle for) or reverse your string and compare with original